### PR TITLE
Added Shing.txt

### PR DESCRIPTION
**Pull request by Matt-Shing-927**

### CODE REVIEW

File: .vscode/shing.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Marc's a ***
```
Comment (AntGa) [inline .vscode/shing.txt:1]: can you define what *** is

Comment (AcousticVibe) [reply .vscode/shing.txt:1]: Reporting this to HR

Comment (tulza) [reply .vscode/shing.txt:1]: 🤨

**Review by AntGa — APPROVED** [review]
LGTM

**Pull request merged**